### ISSUE TITLE
Add POOL as reportable column in qvm-ls

### ISSUE THREAD

**Issue by brendanhoar** [QubesOS/qubes-issues#6831]
### The problem you're addressing (if any)

qvm-ls --running is a quick way for a script to detect running VMs, before performing actions on them. Similarly, adding POOL as a reportable column would be a useful enhancement to discover if a POOL is in use before attempting to perform an action, such as remove, on the POOL.

### Describe the solution you'd like

Add a new column that indicates the pool(s) used by the VM, comma delimited (as a VM may have volumes in different pools).

### Where is the value to a user, and who might that user be?

Users who create temporary pools for certain workflows may utilize multiple pools, some of which are mounted temporarily. These users may want to automate their workflow via scripts. The scripts may need to ask the user to resolve an unexpected pool usage issue before the script can continue. Better to be able to detect the issue before having to catch an error condition.

### Describe any alternative solutions you've considered

Manual checks before removing pools.

### Additional context

-----

**Comment by marmarek** [comment]
There is a catch: a VM may be in several pools at once - for example its
root volume may be in one (the one of its template root volume), private
volume in another and volatile volume in yet another. So, that should
really be separate column for a pool of each volume...
-- 
Best Regards,
Marek Marczykowski-Górecki
Invisible Things Lab

**Comment by brendanhoar** [comment]
> There is a catch: a VM may be in several pools at once - for example its
> root volume may be in one (the one of its template root volume), private
> volume in another and volatile volume in yet another. So, that should
> really be separate column for a pool of each volume...

Indeed, but there also may be more than the four default volumes as well. I thought the comma-delimited approach might be a way to avoid the sprawl, but I'll state my position is neutral to whether the pools are listed in multiple columns by volume or in a single column with a comma-delimited list of unique pools used.

I can see the utility of the one-pool-per-volume w/ multi-columns approach.

Brendan

**Comment by marmarek** [comment]
In practice I think you mostly care about the pool for `private` volume for template-based qubes and for `root` volume for standalone/template. That's why I think separate columns makes sense.

> single column with a comma-delimited list of unique pools used.

Every qube with a dom0-provided kernel has a volume from a linux-kernel pool (kernel modules). That's pretty much useless information most of the time...

**Comment by hippalectryon-0** [comment]
@ben-grande I see you added a tag on January 2026, indicating that this is still a relevant issue - however https://github.com/QubesOS/qubes-issues/issues/5138 is already resolved, can we clarify what work remains to be done here (if any) ?

**Comment by ben-grande** [comment]
> [@ben-grande](https://github.com/ben-grande) I see you added a tag on January 2026, indicating that this is still a relevant issue - however [#5138](https://github.com/QubesOS/qubes-issues/issues/5138) is already resolved, can we clarify what work remains to be done here (if any) ?

No more work, thanks. This is solved. It is a duplicate.

```
qvm-ls --fields NAME,ROOT-POOl,PRIV-POOL
```
